### ISSUE TITLE
Migration Table Install

### ISSUE THREAD

**Issue by tdpdan** [snipe/snipe-it#1619]
Hello,
I am having the same issue described here:
https://github.com/snipe/snipe-it/issues/584

Once I run:
ALTER TABLE companies ENGINE = InnoDB;

Then I re-run the artisan line.

I get: 
SQLSTATE[42S01]: Base table or view already exists: 1050 Table 'migrations' a
lready exists

If I drop the tables and try again, I get the original error:
SQLSTATE[HY000]: General error: 1005 Can't create table 'snipeit_laravel.#sql-53d_43' (errno: 150) (SQL: alter table users add constraint users_company_id_
foreign foreign key (company_id) references companies (id))

Any help would be greatly appreciated!

**Comment by snipe** [comment]
Which artisan line are you trying to re-run?

**Comment by tdpdan** [comment]
php artisan app:install --env=production

**Comment by tdpdan** [comment]
I'm sorry. I hate to bother you, but I really want to use this in our company. It's leaps and bounds beyond any other open source management system. I don't have a lot of experience with ssh, and minimal linux training several years ago.

**Comment by snipe** [comment]
Are you upgrading or installing for the first time? Any info you can provide from here would be really helpful: http://docs.snipeitapp.com/getting-help.html

**Comment by tdpdan** [comment]
- v2.0.5-pre - Released Nov 04, 2015 11:42:37
- Running on a Linux shared hosting cpanel account (Godaddy)
- Installed manually
- Forbidden - You don't have permission to access / on this server. - Additionally, a 403 Forbidden error was encountered while trying to use an ErrorDocument to handle the request.
- Using artisan to install the database via ssh
Everything else in the install has gone smoothly, I'm just botching something here. Best I can tel, everything has the appropriate permissions.

**Comment by tdpdan** [comment]
I deleted everything, created a new ssh key and successfully finished installing. Only other hiccup I had, and may be helpful to others, was the internal 500 error. I wasn't pointing to the public folder of snipe it. Godaddy calls their root level web folder public_html, and I just got thrown off in the wording, translating your very thorough and well written instructions to Godaddy.

Thank you for your help, and your help with your other customers. Between you supporting them in other threads, I was able to get this fantastic product up and running!

Dan
